src/reducers/contacts.reducer.spec.js

```diff
@@ -0,0 +1,147 @@
+import {
+  increaseLikes,
+  decreaseLikes,
+  deleteContact,
+  deleteSelectedContacts,
+  clearContacts,
+  changeIsChecked,
+  fetchFavourites,
+  fetchContacts,
+  addContact
+} from '../actions';
+import contactsReducer from './contacts.reducer';
+import randomContactsList from '../data/data';
+import { getRandomNumber, getFavouritesList } from '../helpers/helper';
+import { DEFAULT_NUMBER_OF_CONTACTS as N } from '../constants/constants';
+
+describe('contacts reducer with id param', () => {
+
+  const state = {
+    contacts: randomContactsList,
```

Comment:
for the sake of a better readability of the code, use some mocked data that is easier to test, for the contacts and for the index